test/test_notebooks.py

```diff
@@ -18,6 +18,14 @@ def append_scrapbook_commands(input_nb_path, output_nb_path, scrap_specs):
         source = "sb.glue(\"{0}\", {1})".format(k, v)
         scrapbook_cells.append(nbf.v4.new_code_cell(source=source))
 
+    # Don't use CDN when running notebook tests
+    for cell in notebook['cells']:
+        if cell.cell_type == 'code' and "ExplanationDashboard(" in cell.source:
+            if "ExplanationDashboard(global_explanation, model, datasetX=x_test)" in cell.source:
```

Comment:
This seems a brittle way of doing this. Would be easier to just have an alternate cell run, but at that point, can we just have a notebook configured for automated testing? I'm not seeing how overwriting a cell in the test is more agreeable than having separate notebooks for testing. Also, I think like 25 should be False, not True.

Reply:
We don't want to have separate notebook, we want to keep using the same notebooks.  We don't want to have duplicate copies of notebooks in the repo, where one is just for testing, that would make updating notebooks difficult.  We already modify the notebook run in tests to add additional cells with validation logic.

"Also, I think like 25 should be False, not True."
Good catch, will update!

Reply:
done, updated!